src/components/SiteVersion.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Link } from '@makerdao/ui-components-core';
+import styled from 'styled-components';
+import { message } from '../static/version.json';
+
+const VersionLink = styled(Link)`
+  font-size: 0.75em;
```

Comment:
at some point we should use official styling for this (using `Text` or at least the typographic presets coming from the design system) but not a blocker to get this merged.